catalog/transformer/catalog-transformer-geojson-queryresponse/pom.xml

```diff
@@ -56,7 +55,10 @@
                 <configuration>
                     <instructions>
                         <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
-                        <Embed-Dependency>json-smart</Embed-Dependency>
+                        <Embed-Dependency>
+                            json-smart,
+                            catalog-core-api-impl;scope=!test
```

Comment:
what does embedding a dependency with not test scope do? i saw it in a few places